ci/example/credentials.example.yaml

```diff
@@ -0,0 +1,9 @@
+---
+# Put here your fill private key
```

Comment:
grammar/sense

Reply:
It makes no sense, but what does?
Fixed.